src/client/src/orangehrmRecruitmentPlugin/pages/ViewActionHistory.vue

```diff
@@ -0,0 +1,351 @@
+import {APIService} from '@/core/util/services/api.service';
+import useDateFormat from '@/core/util/composable/useDateFormat';
+import {formatDate, parseDate} from '@/core/util/helper/datefns';
+import useEmployeeNameTranslate from '@/core/util/composable/useEmployeeNameTranslate';
+import InterviewAttachments from '@/orangehrmRecruitmentPlugin/components/InterviewAttachments.vue';
+import CandidateActionLayout from '@/orangehrmRecruitmentPlugin/components/CandidateActionLayout.vue';
+import InterviewerAutocomplete from '@/orangehrmRecruitmentPlugin/components/InterviewerAutocomplete.vue';
+
+const actionHistoryModel = {
+  id: null,
+  action: {
+    id: null,
+    label: null,
+  },
+  performedBy: {
+    empNumber: null,
+    lastName: null,
+    firstName: null,
+    middleName: null,
+    terminationId: null,
+  },
+  interview: {
+    id: null,
+  },
+  performedDate: null,
+  note: null,
+};
+
+const interviewModel = {
+  interviewName: null,
+  interviewDate: null,
+  interviewTime: null,
+};
+
+export default {
+  components: {
+    'interview-attachments': InterviewAttachments,
+    'candidate-action-layout': CandidateActionLayout,
+    'interviewer-autocomplete': InterviewerAutocomplete,
+  },
+
+  props: {
+    candidateId: {
+      type: Number,
+      required: true,
+    },
+    historyId: {
+      type: Number,
+      required: true,
+    },
+    maxFileSize: {
+      type: Number,
+      required: true,
+    },
+    allowedFileTypes: {
+      type: Array,
+      required: true,
+    },
+  },
+
+  setup(props) {
+    const {locale} = useLocale();
+    const {jsDateFormat} = useDateFormat();
+    const {$tEmpName} = useEmployeeNameTranslate();
+
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      `/api/v2/recruitment/candidates/${props.candidateId}/history`,
+    );
+
+    return {
+      http,
+      locale,
+      jsDateFormat,
+      translateEmpName: $tEmpName,
+    };
+  },
+
+  data() {
+    return {
+      isLoading: false,
+      history: {...actionHistoryModel},
+      interview: {...interviewModel},
+      interviewers: [],
+      rules: {
+        interviewName: [required, shouldNotExceedCharLength(100)],
+        interviewDate: [required, validDateFormat()],
+        interviewTime: [validTimeFormat],
+        interviewerName: [required],
+        note: [shouldNotExceedCharLength(2000)],
+      },
+      statuses: [
+        {id: 1, label: this.$t('recruitment.application_initiated')},
+        {id: 2, label: this.$t('recruitment.shortlisted')},
+        {id: 3, label: this.$t('recruitment.rejected')},
+        {id: 4, label: this.$t('recruitment.interview_scheduled')},
+        {id: 5, label: this.$t('recruitment.interview_passed')},
+        {id: 6, label: this.$t('recruitment.interview_failed')},
+        {id: 7, label: this.$t('recruitment.job_offered')},
+        {id: 8, label: this.$t('recruitment.offer_declined')},
+        {id: 9, label: this.$t('recruitment.hired')},
```

Comment:
@SharukaNP any idea

Reply:
Here we don't call for that API for statuses right? It was for the drop down that I translated the data